docs/zh/user-guide/concepts/why-greptimedb.md

```diff
@@ -0,0 +1,90 @@
+# Why GreptimeDB
+
+GreptimeDB 是一种云原生时间序列数据库，具有分布式、可扩展和高效的特性。关于更多 GreptimeDB 创立背景和所能解决的行业痛点分析，欢迎先阅读[这篇文章](https://mp.weixin.qq.com/s/lTXXsBziCIjJJSCQEW-unw)。
+
+我们的解决方案能更高效地管理海量的时间序列数据，并且能轻松处理时序数据的写入/查询和分析的混合负载需求。无论你的系统是在本地运行还是在云端，GreptimeDB 都可以轻松处理任何规模的负载。基于过去的经验，我们有信心为用户提供任何符合其需求的解决方案和服务。
+
+我们在设计和开发 GreptimeDB 时遵循以下原则：
+
+- **云原生**：从设计 GreptimeDB 的第一天起，其就被定位为一款完全利用云的基础设施和能力来运行和服务的云原生数据库，因此具有出色的可扩展性和容错性。
+- **用户友好**：我们明白开发者体验至关重要，GreptimeDB 的设计关键原则是对开发者友好。这不仅体现在开发、部署和运营方面，在与数据生态系统的无缝兼容性，以及不断完善的文档和社区指南方面，都无一不体现了 GreptimeDB 用户友好的特性。
+- **高性能**：处理时间序列数据的特性之一就是大规模的数据摄取、查询和分析。在高并发场景下依然持续进行性能优化也是我们设计 GreptimeDB 时的核心原则之一。
+- **灵活的架构**：通过抽象良好的分层和封装隔离，GreptimeDB 的部署形式可以满足从嵌入式、单机版、传统集群到云原生的各种环境。
+
+![](https://cdn.jsdelivr.net/gh/filess/img19@main/2023/06/02/1685689928655-5a1f1d0c-70cf-4df0-ba52-6c669062f8ee.png)
```

Comment:
```suggestion
<p><img src="/blogs/2023-06-01-why-greptimedb/image1.png" alt="The architecture of GreptimeDB" style="width: 70%; margin: 0 auto;" /></p>
```

Reply:
prompt: No path exists for "/blogs/2023-06-01-why-greptimedb/image1.png"